### PR TITLE
NOISSUE - Add certificate fields to the Bootstrap service

### PR DESCRIPTION
**Pull request by dborovcanin**
### What does this do?
This pull request adds cert-related fields to Bootstrap service.

### List any changes that modify/break current functionality
None of the existing features is modified.

### Have you included tests for your changes?
Not at the moment. I'll update pull request description as I update the code and tests.

### Did you document any new/modified functionality?
No, I'll update it as I remove `draft` status.

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

**Review by nmarcetic — DISMISSED** [review]

File: bootstrap/postgres/configs_test.go [hunk 1]
```diff
@@ -327,6 +327,62 @@ func TestUpdate(t *testing.T) {
...
+		cert    string
+		certKey string
+		ca      string
+		err     error
+	}{
+		{
+			desc:    "update with wrong owner",
+			key:     "",
+			cert:    "cert",
+			certKey: "crtKey",
```
Comment (manuio) [inline bootstrap/postgres/configs_test.go]: `certKey`

File: bootstrap/postgres/configs_test.go [hunk 2]
```diff
@@ -327,6 +327,62 @@ func TestUpdate(t *testing.T) {
...
+			ca:      "",
+			owner:   "wrong",
+			err:     bootstrap.ErrNotFound,
+		},
+		{
+			desc:    "update a config",
+			key:     c.MFKey,
+			cert:    "cert",
+			certKey: "crtKey",
```
Comment (manuio) [inline bootstrap/postgres/configs_test.go]: certKey

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by drasko — DISMISSED** [review]
LGTM

**Review by manuio — DISMISSED** [review]

**Review by nmarcetic — DISMISSED** [review]

**Review by nmarcetic — APPROVED** [review]

**Review by manuio — APPROVED** [review]

**Pull request merged**